app/views/pages/_form.html.erb

```diff
@@ -34,7 +41,7 @@
     <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible" />
   <% end %>
 
-  <% if question_input.answer_type == "selection" %>
+  <% if question_input.answer_type == "selection" || question_input.answer_type == "file" %>
```

Comment:
If this gets any longer we might want to handle it in a method, but I think this is OK for now!